### ISSUE TITLE
Why use 'nearest' mode rather than 'bilinear' for up-sampling in segmentation mask head?

### ISSUE THREAD

**Issue by wudongming97** [facebookresearch/detr#334]
Hi, 
I have one question about your panoptic segmentation. After getting the attention maps, we need to up-sample them. Could you tell me why to use the mode 'nearest' rather than 'bilinear' in the class MaskHeadSmallConv?

`cur_fpn = self.adapter1(fpns[0])
        if cur_fpn.size(0) != x.size(0):
            cur_fpn = _expand(cur_fpn, x.size(0) // cur_fpn.size(0))
        x = cur_fpn + F.interpolate(x, size=cur_fpn.shape[-2:], mode="nearest")
        x = self.lay3(x)
        x = self.gn3(x)
        x = F.relu(x)`
